docs/package.json

```diff
@@ -26,6 +26,7 @@
   },
   "dependencies": {
     "@docusaurus/core": "2.0.0-beta.20",
+    "@docusaurus/plugin-google-gtag": "^2.1.0",
```

Comment:
Adding a new package will change the pnpm lock file, so we need it updated in the PR.